SocialFilter/sections/specific.txt

```diff
@@ -38,6 +38,7 @@ donbosco-medien.de,giantbomb.com,strefainwestorow.pl,narcity.com,lingohut.com,mt
 aktu.ac.in,fastbikesmag.com,jansatta.com,015.by,actualcomment.ru,airkule.com,akhisarhaber.com,altyazi.net,asyaspor21.tv,bartinolay.com,bedava-film.com,belaruspartisan.by,belgeselciyiz.com,belsat.eu,blog.jetbrains.com,brauer-bund.de,btcnewz.com,businessturkeytoday.com,cj3z.com,corgit.xyz,creatorlabs.net,cwbaltimore.com,daemon-hentai.com,der-postillon.com,desimartini.com,discoverskills.com,discoverychannel.com.tr,dizilara.org,dizitube.net,dizivay.com,earthsky.org,efelerhaber.com,egeyonhaber.com,elazigyenihaber.com,ellegirl.ru,erkekadam.org,erzurumajans.com,eurolab.ua,filmakinesi.net,filmold.com,fontanka.ru,forums.flyer.co.uk,fox38corpuschristi.com,fox47.com,foxkansas.com,foxrichmond.com,gazetaby.com,gercekizmir.com,gidavitrini.com.tr,govoritmoskva.ru,halklailiskiler.com.tr,hentairules.net,hizlifilmizlehd.com,hornygamer.com,inforesist.org,jeotermalhaberler.com,jetvis.ru,kachelmannwetter.com,karsolay.com,kbsi23.com,kdsm17.com,kevinpurcell.org,kg-portal.ru,kinopoisk.userecho.com,kp.ua,larende.com,losst.ru,magazinekibi.com,manisamanset.gen.tr,mforos.com,mogilev.online,muchohentai.com,multifilemirror.com,mvs.gov.ua,mytuner-radio.com,mywebtimes.com,namemc.com,news-journalonline.com,nk.org.ua,nkj.ru,noticierodigital.com,ntp-servers.net,obltv.ru,orbita.dn.ua,pctip.ru,podrobnosti.ua,ppt.ru,proglang.su,queenbet4.tv,raider.io,raleighcw.com,real-vin.com,rmj.ru,rusvesna.su,sanatatak.com,seirsanduk.us,simdifilmizle.com,sonkaleizmir.com,sp-mobile-blog.com,star64.tv,tayga.info,tomiks33.ru,torrentbe.net,torrentrg.org,tqm.com.ua,tribundergi.com,tva.in.ua,utofiles.com,washingtontimes.com,wcti12.com,womanel.com.ua,wpgh53.com,wtto21.com,wutv29.com,xn----8sbcm1bmcar0m.xn--p1ai##.twitter-timeline
 !
+blog.doist.com##div[class$="toc-share"]
```

Comment:
`blog.doist.com##.db-actionbar__toc-share` is more reliable

Reply:
and please make rules for newsletters and social buttons (in articles)

Reply:
@Sergey-Lyapin You're someone who can reproduce everything. I can't reproduce anything else. I wonder how you do it.

Reply:
I don't know, probably some experience =) https://github.com/AdguardTeam/AdguardFilters/commit/8f6396bbdfbbde66dc083678916eaea48f5b3566
![Screenshot 2021-04-12 at 13 01 50](https://user-images.githubusercontent.com/18536328/114378530-7bd3b500-9b90-11eb-987b-00e5c7e245fd.png)
![Screenshot 2021-04-12 at 13 02 06](https://user-images.githubusercontent.com/18536328/114378541-7e360f00-9b90-11eb-8a81-03f5701526a5.png)
![Screenshot 2021-04-12 at 13 02 48](https://user-images.githubusercontent.com/18536328/114378547-7fffd280-9b90-11eb-8f8d-d6b844e3e070.png)
![Screenshot 2021-04-12 at 13 09 19](https://user-images.githubusercontent.com/18536328/114378557-81c99600-9b90-11eb-9e7f-881e746bbbf9.png)